src/pages/species/components/species-list/styles.scss

```diff
@@ -18,6 +18,7 @@
     &.__active{
       font-size: 30px;
       height: 70px;
+      background-color: #313131;
```

Comment:
move it to settings color variables? you can use http://chir.ag/projects/name-that-color this handy page to name colors